modepy/matrices.py

```diff
@@ -131,6 +131,12 @@ def multi_vandermonde(
     A sequence of matrices is returned--i.e. this function
     works directly on :func:`modepy.Basis.gradients` and returns
     a tuple of matrices.
+
+    .. note::
+
+        Depending on the use case, it may be convenient to instead call
+        :func:`vandermonde` with the result of :meth:`Basis.derivatives`,
+        which is mathematically equivalent.
```

Comment:
Maybe phrase this more like "it also works with Basis.derivatives"? It's a bit unclear what use cases could / should work.

Reply:
Did some rephrasing, though I'm not sure I covered what you asked for here. This is in the docstring for `multi_vandermonde`, which won't work with `Basis.derivatives`. I did specify the use case more clearly. LMK if this works.

Reply:
Yeah, that's a lot clearer to me. Thanks!